examples/data-objects/monaco/src/dataObject.ts

```diff
@@ -0,0 +1,40 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+// inspiration for this example taken from https://github.com/agentcooper/typescript-play
+import { DataObject } from "@fluidframework/aqueduct";
+import { IFluidHandle } from "@fluidframework/core-interfaces";
+import { SharedString } from "@fluidframework/sequence";
+
+/**
+ * Component for using the Monaco text editor.
+ */
+export class MonacoRunner extends DataObject {
+    private _text: SharedString | undefined;
```

Comment:
Nice, its immediately clear what the concerns of this DataObject are. At some point it would be interesting to explore if we can do this with less boilerplate and bind our DataObject in more declarative way.

Reply:
Yeah - there might also be nicer interfaces rather than handing out the SharedString directly, it would require some deeper investigation into the Monaco integration.